src/main/CMakeLists.txt

```diff
@@ -350,7 +350,12 @@ if (OS_IS_WIN)
             )
 
     if (SNDFILE_LIB)
-        install(FILES ${DEPENDENCIES_DIR}/libsndfile-1.dll DESTINATION bin)
+        if (CC_IS_MINGW)
+            install(FILES ${SNDFILE_LIB} DESTINATION bin)
+        else()
+            find_library(dll_sndfile NAMES "sndfile" "libsndfile-1" PATHS ${DEPENDENCIES_DIR} NO_DEFAULT_PATH)
```

Comment:
We have /build/cmake/FindSndFile.cmake, I believe, find should be there

Reply:
Even before my change we apparenerly don't (fully) rely on FindSndFile.cmake (which, as mentioned, for MSVC finds the .lib file, not the .dll), but rather hardcode it.

Reply:
I got it, thanks